tensorflow/python/keras/layers/convolutional.py

```diff
@@ -127,6 +138,11 @@ def __init__(self, rank,
     if filters is not None and not isinstance(filters, int):
       filters = int(filters)
     self.filters = filters
+    self.groups = groups
+    if filters is not None and filters % self.groups != 0:
```

Comment:
what if `self.groups = None`? Should we allow that and make None == 1?

Reply:
I don't think this case will come up very much in practice, but it is very easy to support. I added a quick check in d74951f42d8ae8b7c29fb428c3745c90196f95bc